utils/get/get.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * @param obj - an object that needs to be safely traversed `{ a: { b: true } }`
+ * @param path - a string that represents the path of the object `'a.b'`
+ * @param defaultValue - a value to return if the path doesn't resolve in the object
+ */
+export const get = (obj: any, path: string, defaultValue?: any) => path.split('.')
```

Comment:
@zero0Halo bc I unit tested this, I noticed that I omitted the default being an optional prop. thanks for the push!

Reply:
One last change. Put a a description as to what the function is for on line 2. I am a comment nazi. Plain english, ex: "Our version of lodash's get() method, just less bloated" or something to that effect.

Reply:
Nice catch.

Reply:
Looks good!